protos/remote_id/remote_id.proto

```diff
@@ -0,0 +1,125 @@
+syntax = "proto3";
+
+package mavsdk.rpc.remote_id;
+
+import "mavsdk_options.proto";
+
+option java_package = "io.mavsdk.remote_id";
+option java_outer_classname = "RemoteIdProto";
+
+service RemoteIdService {
+    /*
+     * Update the BasicId structure sent with the basic_id packet
+     */
+    rpc SetBasicId(SetBasicIdRequest) returns(SetBasicIdResponse) { option (mavsdk.options.async_type) = SYNC; }
+    /*
+     * Update the Location structure sent with the location packet
+     */
+    rpc SetLocation(SetLocationRequest) returns(SetLocationResponse) { option (mavsdk.options.async_type) = SYNC; }
+    /*
+     * Update the System structure sent with the system packet
+     */
+    rpc SetSystem(SetSystemRequest) returns(SetSystemResponse) { option (mavsdk.options.async_type) = SYNC; }
+    /*
+     * Update the OperatorId structure sent with the operator_id packet
+     */
+    rpc SetOperatorId(SetOperatorIdRequest) returns(SetOperatorIdResponse) { option (mavsdk.options.async_type) = SYNC; }
+    /*
+     * Update the SetSelfId structure sent with the self_id packet
+     */
+    rpc SetSelfId(SetSelfIdRequest) returns(SetSelfIdResponse) { option (mavsdk.options.async_type) = SYNC; }
+}
+
+message SetBasicIdRequest {
+    BasicId basic_id = 1; // Desired basic_id
+}
+message SetBasicIdResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message SetLocationRequest {
+    Location location = 1; // Desidered location
+}
+message SetLocationResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message SetSystemRequest {
+    SystemId system = 1; // Desidered system
+}
+message SetSystemResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message SetOperatorIdRequest {
+    OperatorId system = 1; // Desidered operator_id
+}
+message SetOperatorIdResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message SetSelfIdRequest {
+    SelfId self_id = 1; // Desidered self_id
+}
+message SetSelfIdResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message BasicId {
+    uint32 id_type = 1; // Indicates the format for the uas_id field of this message.
+    uint32 ua_type = 2; // Indicates the type of UA (Unmanned Aircraft).
+    string uas_id = 3; // UAS ID following the format specified by id_type.
+}
+
+message Location {
+    uint32 status = 1; // Indicates whether the unmanned aircraft is on the ground or in the air.
+    uint32 direction_deg = 2; // Direction over ground measured clockwise from true North: 0 - 359 deg.
+    uint32 speed_horizontal_cm_s = 3; // Ground speed. Positive only. cm/s
+    int32 speed_vertical_cm_s = 4; // The vertical speed. Up is positive. cm/s
```

Comment:
Thanks for the changes :blush:.

Is there a reason to have this in `cm/s` instead of `m/s`?

Reply:
laziness I think :flushed:. 
I used the same units presents in  mavlink packets. 
I agree that is better to use SI units and let he plugin handle the conversions.